src/main/java/uk/gov/pay/products/persistence/dao/ProductDao.java

```diff
@@ -116,4 +116,32 @@ private List<ProductUsageStat> findProductsAndUsageQuery(Integer gatewayAccountI
         }
         return queryBuilder.getResultList();
     }
+
+    public List<ProductUsageStat> findUnusedProducts(Integer gatewayAccountId) {
+        return findUnusedProductsQuery(gatewayAccountId);
+    }
+
+    public List<ProductUsageStat> findUnusedProductsQuery(Integer gatewayAccountId) {
+        boolean shouldFilterGatewayAccount = gatewayAccountId != null;
+
+        String conditionalFilter = shouldFilterGatewayAccount ?
+                "AND product.gateway_account_id = ?1 " :
+                "";
+
+        String query = "SELECT product.* FROM products product " +
+                "LEFT JOIN payments payment ON product.id = payment.product_id " +
+                "WHERE payment.product_id IS NULL " +
+                conditionalFilter;
+
+        var queryBuilder = entityManager.get()
+                .createNativeQuery(query, ProductEntity.class);
+
+        if (shouldFilterGatewayAccount) {
+            queryBuilder = queryBuilder.setParameter(1, gatewayAccountId);
+        }
+
+        List<ProductEntity> products = queryBuilder.getResultList();
+
+        return products.stream().map(product -> new ProductUsageStat(0L, null, product)).toList();
```

Comment:
these can be combined
```suggestion
        return queryBuilder.getResultList()
            .stream()
            .map(product -> new ProductUsageStat(0L, null, (ProductEntity) product))
            .toList();
```